cypress/integration/seededFlows/adminFlows/users/manageCredits.spec.js

```diff
@@ -6,9 +6,11 @@ function openCreditsModal() {
   return cy.getModal();
 }
 
-function closeUserUpdatedMessage(message) {
+function verifyAndDismissUserUpdatedMessage(message) {
```

Comment:
I changed these function names just since with the new assertion in there it felt like it was doing a bit more than closing the message now